src/components/MediaViewer/MainMediaDisplay.js

```diff
@@ -73,7 +73,7 @@ const MainMediaDisplay = ( {
             ? (
               <View className="absolute bottom-4 right-4">
                 <TransparentCircleButton
-                  onPress={( ) => onDeletePhoto( uri )}
+                  onPress={( ) => onDeletePhoto( photo.localFilePath )}
```

Comment:
This is not going to work when trying to delete a remote photo, e.g. one that was uploaded to the website and downloaded in the app.